website/cue/reference/components/sources/host_metrics.cue

```diff
@@ -585,6 +585,10 @@ components: sources: host_metrics: {
 	}
 
 	telemetry: metrics: {
-		processed_events_total: components.sources.internal_metrics.output.metrics.processed_events_total
```

Comment:
Has `processed_events_total` already been deprecated/removed from this source or was this accidentally removed?

Reply:
When you look into [the `host_metrics` internal events file](https://github.com/vectordotdev/vector/blob/master/src/internal_events/host_metrics.rs), there is no `emit_events` so no `processed_events_total` 🤔

Reply:
Gotcha, seems like it was an outdated doc then.

Reply:
`component_sent_events_total ` is actually aliased as `processed_events_total` by the metrics recorder:

https://github.com/vectordotdev/vector/blob/86bfec8d70cb30158964a0208d37b30723be3e8f/lib/vector-core/src/metrics/mod.rs#L144-L151

For sources, `component_sent_events_total` is published by the topology:

https://github.com/vectordotdev/vector/blob/86bfec8d70cb30158964a0208d37b30723be3e8f/src/source_sender/mod.rs#L202-L253

So this component should still actually be publishing this deprecated metric.

Reply:
I realize it is a bit tedious, but I'd recommend as you go through these, to actually run the sources locally and observe that the metrics being reported are accurate.